client/src/app.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.stat={
+
+    };
+  }
+
+  render() {
+    return(
+
+      <div> Hi This is a message</div>
+    );
+  }
```

Comment:
I believe the style guide asks us to end this with a semi-colon. Please check
```suggestion
   };
```